docs/code-snippets/office-snippets.yaml

```diff
@@ -3356,6 +3356,52 @@ Office.Settings#saveAsync:member(2):
     function write(message){
         document.getElementById('message').innerText += message; 
     }
+Office.Tab#id:member:
```

Comment:
Doesn't need to be done in this PR, but consider also adding the snippet to [Office.ribbon.requestUpdate](https://learn.microsoft.com/javascript/api/office/office.ribbon#office-office-ribbon-requestupdate-member(1)) and [Office.RibbonUpdaterData](https://learn.microsoft.com/javascript/api/office/office.ribbonupdaterdata).

Reply:
> Doesn't need to be done in this PR, but consider also adding the snippet to [Office.ribbon.requestUpdate](https://learn.microsoft.com/javascript/api/office/office.ribbon#office-office-ribbon-requestupdate-member(1)) and [Office.RibbonUpdaterData](https://learn.microsoft.com/javascript/api/office/office.ribbonupdaterdata).

Done